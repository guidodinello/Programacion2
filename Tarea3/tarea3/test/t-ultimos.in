# Prueba de tiempo sumaUltimosPositivos
tiempo_sumaUltimosPositivos 10000 20000 5 10000 5
Fin
