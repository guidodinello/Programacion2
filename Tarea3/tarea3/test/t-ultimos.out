1>#  Prueba de tiempo sumaUltimosPositivos.
2>
 Construyendo el Ã¡rbol. 
 Obteniendo suma ultimos positivos. 
 Liberando binario. 
3>Fin.
